src/app/login/login.component.html

```diff
@@ -0,0 +1,24 @@
+<div id='description-container'>
+	<div id='description'>
+		Team Mkr is a platform to connect with other developers.
+		Find others who want to work on similar skills or projects
+		and start projects with them.
+	</div>
+</div>
+<form id='login-box'>
+	<div id='content'>
+		<h2>Log in to Team Mkr</h2>
```

Comment:
Semantically, this should probably be an h1 since its the most important statement on the page